keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/ResourceMetadata.kt

```diff
@@ -22,6 +22,7 @@ import de.huxhorn.sulky.ulid.ULID
 data class ResourceMetadata(
   val name: ResourceName,
   val uid: UID,
+  val serviceAccount: String,
```

Comment:
Is this OSS compatible?

Reply:
Ya, this is just good old fiat service accounts.